### PR TITLE
Updated the PRs as to the Comments

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 1]
```diff
@@ -1,18 +1,18 @@
 /*
  * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
+*
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Some formatting issue in the license header.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 2]
```diff
@@ -397,6 +401,7 @@ public void addRole(String roleName) throws MalformedURLException, XPathExpressi
      * @throws XPathExpressionException
      */
     public void assignRoleToUser(String[] userNames) throws MalformedURLException, XPathExpressionException {
+
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Remove this line.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 3]
```diff
@@ -477,7 +482,8 @@ public boolean isResourceExist(String resourcePath) {
      * @throws InterruptedException
      */
     public void deleteDashboards() throws MalformedURLException, XPathExpressionException, InterruptedException {
-        redirectToLocation("portal", "dashboards");
+
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Remove this line.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 1 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the newest commit

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 2 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the newest commit

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 3 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the newest commit

**Pull request merged**